Koronavilkku/Koronavilkku/View/Exposures/ExposuresLastCheckedView.swift

```diff
@@ -66,7 +74,7 @@ class ExposuresLastCheckedView: UIView {
     }
 
     private func bindViewModel() {
```

Comment:
After prior change on line 51, this private method is no longer used. Can be removed